<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CompressMenu</class>
 <widget class="QFrame" name="CompressMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>675</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(110, 110, 100);
color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="labelSourcePath">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>651</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>图像源路径：D:/Example/Image/</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTitle">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>380</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>图像压缩</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonOutputPath">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(253, 195, 99);
color: rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>修改输出路径</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelImageAmount">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>381</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>图像数量：5张</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelIcon">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>0</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/IconPNG/Icon/Compress-3780.PNG);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Line" name="line1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>571</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="labelStart">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>526</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>开始执行</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCompressTitle">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>381</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>压缩选项：</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelOutputPath">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>290</y>
     <width>651</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>输出路径：D:/Example/Image/</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonStart">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>490</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/IconPNG/Icon/Start-3780.PNG);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelExit">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>626</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>返回首页</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxPercent">
   <property name="geometry">
    <rect>
     <x>215</x>
     <y>485</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;等线&quot;;</string>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>50</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonExit">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>590</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/IconPNG/Icon/Exit-3780.PNG);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelInfoTitle">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>85</y>
     <width>381</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>基本信息：</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPercent">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>490</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCompressPercent">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>490</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>压缩至原图分辨率的：</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelStateOriginal">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>580</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>原图的分辨率：6000 * 4000</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelStateCompress">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>620</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>压缩后分辨率：3000 * 2000</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelStateTitle">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>540</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>以第一张图片为例：</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelButtonPixel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>430</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>按像素压缩</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonPixel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>388</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/IconPNG/Icon/CompressPixel-3780.PNG);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelButtonPercent">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>430</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>按百分比压缩</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonPercent">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>388</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/IconPNG/Icon/CompressPercent-3780.PNG);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelPxH">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>551</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>px</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxHeight">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>548</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;等线&quot;;</string>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>100000</number>
   </property>
   <property name="singleStep">
    <number>100</number>
   </property>
   <property name="value">
    <number>200</number>
   </property>
  </widget>
  <widget class="QLabel" name="labelPxW">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>504</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>px</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelWidth">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>506</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>水平：</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelHeight">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>554</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>垂直：</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxWidth">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>500</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;等线&quot;;</string>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>100000</number>
   </property>
   <property name="singleStep">
    <number>100</number>
   </property>
   <property name="value">
    <number>300</number>
   </property>
  </widget>
  <widget class="QLabel" name="labelWarning">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>620</y>
     <width>591</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>警告：该模式下若原始图片的比例不符合设置的比例，可能会导致图像变形！</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
